runtime/src/execute_cost_table.rs

```diff
@@ -106,6 +106,12 @@ impl ExecuteCostTable {
             }
         }
 
+        // Logs events of unusual input `value` or `aggregated.ema`
+        const MAX_COST = 200_000u64;
+        if value > MAX_COST || aggregated.ema > MAX_COST as f64 {
+            debug!("program {:?} cost {} aggregated {:?}", key, value, aggregated);
```

Comment:
maybe run this version on our node with `debug` level; Also would capture this debug log line that can help reconstruct events: https://github.com/solana-labs/solana/blob/master/core/src/cost_update_service.rs#L174-L177